README.md

```diff
@@ -91,112 +128,153 @@ To compile this plugin, you need:
-* `core_power`
+* `CORE/P`
+    or old syntax: `core_power`
 
     Collect power consumption information for every core.
 
-* `gpu_energy`
+* `GPU/E`
+    or old syntax: `gpu_energy`
 
     Collect power consumption information for every gpu.
 
-* `gpu_power`
+* `GPU/P`
+    or old syntax: `gpu_power`
 
     Collect power consumption information for every gpu.
 
-* `dram_energy`
+* `DRAM/E`
+    or old syntax: `dram_energy`
 
     Collect power consumption information for every dram.
 
-* `dram_power`
+* `DRAM/P`
+   or old syntax: `dram_power`
 
     Collect power consumption information for every dram.
 
-E.g.:
+* `PACKAGE0/E`, etc.
 
-    export SCOREP_METRIC_X86ENERGY_PLUGIN="gpu_power"
+    Collect power consumption information for package on node 0.
+    This also works with every other metric.
 
-or for VampirTrace:
+E.g.:
 
-    export VT_PLUGIN_CNTR_METRICS=X86ENERGY_*_energy
+    export SCOREP_METRIC_X86_ENERGY_PLUGIN="GPU/P" or
+    export SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN="GPU/P"
 
 ### Environment variables
 
-* `SCOREP_X86ENERGY_INTERVAL_US`/`VT_X86ENERGY_INTERVAL_US` (default=100000)
+* `SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN`/`SCOREP_METRIC_X86_ENERGY_PLUGIN`
 
-    The time in usecs, between two consecutive reads of the power/energy values. A longer interval
-    means less disturbance, a shorter interval is more exact.
+    Comma-separated list of sensors, and physical quantity. Can also contain wildcards, e.g.
+    `BLADE/E,*/E`
 
-    On Intel CPUs, the registers are updated roughly every msec. If you choose an interval of 1ms
-    you might find highly variating power consumptions. To gain most exact values, you should set
-    the amplitude to 10, if you can live with less accuracy, you should set it to 100000.
+* `SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN_VERBOSE`/`SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN_VERBOSE` (default `WARN`)
 
-* `SCOREP_X86ENERGY_CPU_PIN`/`VT_X86ENERGY_CPU_PIN` (default=no pinning)
+    Controls the output verbosity of the plugin. Possible values are: `FATAL`, `ERROR`, `WARN`,
+    `INFO`, `DEBUG`. If set to any other value, `DEBUG` is used. Case in-sensitive.
 
-    The CPU to pin the register reading thread to. The reading thread uses the `msr` files of core
-    `<X86_ENERGY_PLUGIN_CPU_PIN>` of each package (default: the first core of a socket). You might
-    consider to set it to a free Hyperthread.
+* `SCOREP_METRIC_X86_ENERGY_SYNC_PLUGIN_OFFSET` (default `70000.0`)
 
-* `SCOREP_X86ENERGY_BUF_SIZE`/`VT_X86ENERGY_BUF_SIZE` (default=4M)
+    Can be set to any double variable. Defines the offset in mW that is added to 
+    the blade, because with RAPL only the energy consumption of the
+    CPU is measurable. Blade is the sum of all RAPL values excepted the core
+    because there are included in package.
+    The default value of 70000.0 mW was measured on Taurus
 
-    The size of the buffer for storing samples. Can be suffixed with G, M, and K.
+* `SCOREP_X86_ENERGY_PLUGIN_READING_TIME` (default `5`)
```

Comment:
Because it is not an interval. If a measurement takes longer than 5 ms we assume that the result is no longer reliable, and therefore 0 is reported.

Through I am not sure this is needed with RAPL. It was introduced with the HDEEM plugin.